### DISCUSSION TITLE
What is wrong with my babel config?

### DISCUSSION THREAD

**Discussion by ariannargesi** [vercel/next.js#63690] [Help]
### Summary

I'm looking in repositories under https://github.com/vercel/next.js/tree/canary/examples and In many config files, I get the following error regard to babel 

![Screenshot from 2024-03-26 01-20-56](https://github.com/vercel/next.js/assets/46089900/54dd29d5-50fa-4acb-8a4c-6a2cf982e997)
![Screenshot from 2024-03-26 01-20-43](https://github.com/vercel/next.js/assets/46089900/f9eb55b6-8820-4f04-b4dd-f7eb94122d32)
![Screenshot from 2024-03-26 01-20-23](https://github.com/vercel/next.js/assets/46089900/459fb2b8-9bd1-4ea4-b268-8be03f5b9d77)

**Comment by SouthLink** [comment]
Try creating a.babelrc file in the root of your project.
```
{
  "presets": [
    "next/babel"
  ]
}
```
Then change the contents of the.eslintrc.json file in the root directory to :

```
{
  "extends": ["next/core-web-vitals", "prettier"],
}
```
This should help you.
